nitrite.go

```diff
@@ -239,7 +240,11 @@ func Verify(data []byte, options VerifyOptions) (*Result, error) {
 		return nil, ErrBadAttestationDocument
 	}
 
-	if "" == doc.ModuleID || "" == doc.Digest || 0 == doc.Timestamp || nil == doc.PCRs || nil == doc.Certificate || nil == doc.CABundle {
+	if "" == doc.ModuleID ||
```

Comment:
Changing to fit on one line and taking out the test `nil == doc.CABundle` since CABundle is nil when building an attestation of a self-signed cert in https://github.com/blocky/nitriding/blob/f11d71cf795d84cdf3c11d104e258fc3ac38851b/attester.go#L140

Reply:
@mwittie, using the options turned out way better!